### PR TITLE
Unit test for the single speed/stage IEER calculation

### LINKED ISSUE #89: Complete Test Cases for DX systems
To Do Task: Add error test cases for DX system operation.

### LINKED ISSUE #91: Dx curve file
Hi @lymereJ, I guess we have a duplicate DX curve file here?
copper/data/unitary_dx_curves.json

### LINKED ISSUE #96: Unitary DX Issues
- [ ] `compressor_speed` is not assigned, see [here](https://github.com/pnnl/copper/blob/fe18acd7708deb10cd13c79cc27ee32efb8eb57e/copper/unitarydirectexpansion.py#L28C9-L28C25)
- [ ] there's not units for the capacity, users might want to specify the capacity using different units, just like for chillers

### PR DESCRIPTION
**Pull request by wanhanlong1130**
new json file and new unit tests for DX.

### CODE REVIEW

**Comment by lymereJ** [comment]
@wanhanlong1130 - Could you please do a reverse merge of the `develop` branch into your feature branch? It looks like there are a few conflicts.

**Comment by lymereJ** [comment]
I've linked issues that this PR should address. Let's make sure that they're addressed before merging it!

**Comment by lymereJ** [comment]
@aowabinr can you please review the library file brought over from #95? Thanks!

**Review by lymereJ — CHANGES_REQUESTED** [review]
@wanhanlong1130 - Could you please do a reverse merge of the develop branch into your feature branch? It looks like there are a few conflicts.

File: copper/data/unitarydirectexpansion_curves.json [hunk 1]
```diff
@@ -0,0 +1,3992 @@
...
+        "model": "simplified_bf",
+        "condenser_type": "air",
+        "compressor_type": "scroll",
+        "compressor_speed": "variable",
+        "ref_cap": 20221.9016703,
+        "ref_cap_unit": "W",
+        "full_eff": 12.0,
+        "full_eff_unit": "cop",
+        "sim_engine": "energyplus",
+        "full_eff_ref_std": "ahri_341/361",
```
Comment (lymereJ) [inline copper/data/unitarydirectexpansion_curves.json]: I think that we're using AHRI Standard 340/360.

File: copper/data/unitarydirectexpansion_curves.json [hunk 2]
```diff
@@ -0,0 +1,3992 @@
...
+        "ref_gross_cap_unit": "si",
+        "ref_net_cap_unit": "si"
+    },
+    "48_50A040": {
+        "eqp_type": "UnitaryDirectExpansion",
+        "source": "Carrier",
+        "model": "simplified_bf",
+        "condenser_type": "air",
+        "compressor_type": "scroll",
+        "compressor_speed": "constant/variable",
```
Comment (lymereJ) [inline copper/data/unitarydirectexpansion_curves.json:1148]: @aowabinr - Is this supposed to be variable capacity? Variable capacity compressors, I think, run a constant speed but have the capacity to modulate their capacity.

**Comment by wanhanlong1130** [comment]
> @wanhanlong1130 - Could you please do a reverse merge of the develop branch into your feature branch? It looks like there are a few conflicts.

Already done!

File: copper/data/unitarydirectexpansion_curves.json (hunk 1 above)
Comment (wanhanlong1130) [reply copper/data/unitarydirectexpansion_curves.json]: May need Aowabin to update from his side as well.

Comment (lymereJ) [reply copper/data/unitarydirectexpansion_curves.json]: @aowabinr - FYI.

**Review by lymereJ — APPROVED** [review]

**Pull request merged**